docker-images/openemr/5.0.0/run_openemr.sh

```diff
@@ -89,5 +118,7 @@ fi
 # ensure the auto_configure.php script has been removed
 rm -f auto_configure.php
 
+echo "Starting cron daemon!"
+crond
```

Comment:
What's going on here? Is certbot using cron in a non-obvious way?

Reply:
it's using it in the obvious way. This is just starting the cron daemon

Reply:
What's using the cron daemon, though? We're not maintaining any crontabs ourselves.

Reply:
oh whoops. I forgot to actually pipe the echo on line 20 anywhere. It runs the renewal script every day to check if certs need to be renewed. Fixing shortly...